src/main/resources/application.yml

```diff
@@ -1,26 +1,39 @@
 spring:
+  profiles: 
+    active: devqa
   application:
     name: platform.training.bookmiddle
     version: 1.0.0
   jackson:
     default-property-inclusion: non_null
-
+    
 server:
-  port: ${service.port:8080}
+  port: ${service.port:8888}
 
 mongoDB:
   database: bookDB
   host: 127.0.0.1
   port: 27017
   
-sqs:
-  queueUrl: https://sqs.us-east-2.amazonaws.com/404669482207/book-track.fifo
```

Comment:
I believe this is Diego's URL, you should keep it when it isn't a test.

Reply:
Actually, as Darren mentioned we are not going to use Fifo Queue, so we need to update it anyway

Reply:
OK, But leave that to another specific commit. I don't think that we will change it.